portlets/chat-video-portlet/docroot/js/inject_chat_portlet.js

```diff
@@ -705,9 +720,7 @@ AUI().use(
 
 										// Hang up control button
 
-										if (state === State.STOPPED || state === State.STOPPING ||
-											state === State.DELETED || state === State.DELETING) {
-
+										if (state === State.STOPPED || state === State.STOPPING || state === State.DELETED || state === State.DELETING) {
```

Comment:
We should probably cache all these ```State.STOPPED``` as variables since we are using them in quite a few places.